### ISSUE TITLE
Note Encryption

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/best-evidence#140]
TBC at Sprint Planning

**Comment by SimonLab** [comment]
linked to #149 At the moment the database is protected by password and is not accessible, users can only access their own information.
The wording of the consent page as been updated to reflect this.

**Comment by markwilliamfirth** [comment]
@ajburls just wanted you to confirm in writing on GitHub if note encryption is or is not something you may want in future (as it was in the original scope) so as to decide whether to officially close this or leave it in the backlog?

**Comment by ajburls** [comment]
Yes it is something we will want to consider later but relatively low priority for the time being, I have labelled it priority 4, as there are other things we need before that.
